### PR TITLE
Documentation Update

### PR DESCRIPTION
**Pull request by Chris92de**
- Slight theme changes
  -> Switched color palette to green
  -> Added social links in footer
  -> Added Copyright notice in footer
  -> Added logo in header
- Added initial articles from the Trackmania Documentation project
- Added top navbar
- Changed docs structure to be more clear
- Moved navigation declaraction to mkdocs.yml
- Added repo link in header

### CODE REVIEW

File: docs/play/what-are-arcade-channels.md
Comment (TonaTM) [inline docs/play/what-are-arcade-channels.md:1]: Is there a link to the guide?

File: docs/play/what-is-a-seasonal-campaign.md
Comment (TonaTM) [inline docs/play/what-is-a-seasonal-campaign.md:1]: The first boxes are weird, is it a mistake? :D

File: docs/club/activities/skin-uploads.md
[the remark this answers is not in the document]
Comment (Chris92de) [reply docs/club/activities/skin-uploads.md:1]: Added box, will push soon.

File: docs/create/map-review/submit-your-map-to-map-review.md
[the remark this answers is not in the document]
Comment (Chris92de) [reply docs/create/map-review/submit-your-map-to-map-review.md:1]: Will rework formatting on this article and add this information. Coming in next push tomorrow.

File: docs/play/what-are-arcade-channels.md
Comment (Chris92de) [reply docs/play/what-are-arcade-channels.md:1]: Ah, no, I wanted to do it but forgot. Adding this for the next push.

File: docs/play/what-is-a-seasonal-campaign.md
Comment (Chris92de) [reply docs/play/what-is-a-seasonal-campaign.md:1]: Yes, mistake, had tried something and forgot to remove. Gone in next update!

File: docs/club/activities/room.md
Comment (TonaTM) [inline docs/club/activities/room.md:1]: In Create aRoom activity, please specify that dedicated servers are for PC players only and console player will not be able to join them

Comment (TonaTM) [reply docs/club/activities/room.md:1]: The https://wiki.trackmania.io/en/dedicated-server/Setup/Windows page is maybe something we can add to this documentation, what do you think about that?

**Pull request merged**